projects/core/directives/dropdown/dropdown-open-legacy.directive.ts

```diff
@@ -10,11 +10,19 @@ import {BehaviorSubject} from 'rxjs';
         '[tuiDropdownOpen]:not([tuiDropdown]),[tuiDropdownOpenChange]:not([tuiDropdown])',
 })
 export class TuiDropdownOpenLegacy {
+    private readonly openStateSub = new Subject<boolean>();
+
     @Output()
-    public readonly tuiDropdownOpenChange = new BehaviorSubject(false);
+    public readonly tuiDropdownOpenChange = this.openStateSub
+        .asObservable()
```

Comment:
Unnecessary, pipe would already turn it into Observable

Reply:
Thanks for pointing that out.
Resolved!